website/addons/googledrive/views/config.py

```diff
@@ -21,9 +20,11 @@
 @must_have_permission(permissions.WRITE)
```

Comment:
@sloria, @lyndsysimon,

We're getting closer and closer to having converting class-based views to a very easy job. Perhaps some of the assessment done in @brianjgeiger 's API proposal will facilitate this process.